src/test/scala/nl/knaw/dans/lib/logging/servlet/masked/MaskerSpec.scala

```diff
@@ -122,4 +118,62 @@ class MaskerSpec extends FlatSpec with Matchers {
     Masker.formatAuthenticationParameter("other-header" -> Seq("some value")) shouldBe
       "other-header" -> Seq("some value")
   }
+
+  private case class TestCase(address: String, expected: String)
```

Comment:
Have you looked at http://www.scalatest.org/user_guide/table_driven_property_checks. Probably that is exactly what you're trying to achieve here 😉

Reply:
If only they would fix https://github.com/scalatest/scalatest/issues/178

Reply:
Tried version 3.0.5 of scalatest, no improvement on this point.